shared/js/ui/views/top-blocked-truncated.es6.js

```diff
@@ -9,6 +9,7 @@ function TruncatedTopBlocked (ops) {
   Parent.call(this, ops)
 
   this.model.getTopBlocked().then(() => {
+    setTimeout(() => this.rerenderList(), 750)
```

Comment:
Note: I tried setting a timeout on `this._rerender()`, but it was breaking clicking on "all trackers" to go to the subview